sci-chemistry/tinker/tinker-8.2.1-r1.ebuild

```diff
@@ -1,31 +1,33 @@
-# Copyright 1999-2022 Gentoo Authors
+# Copyright 1999-2023 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=8
 
-inherit flag-o-matic fortran-2 java-pkg-opt-2 toolchain-funcs
+inherit flag-o-matic fortran-2 java-pkg-2 toolchain-funcs
 
 DESCRIPTION="Molecular modeling package that includes force fields, such as AMBER and CHARMM"
-HOMEPAGE="http://dasher.wustl.edu/tinker/"
-SRC_URI="http://dasher.wustl.edu/${PN}/downloads/${P}.tar.gz"
+HOMEPAGE="https://dasher.wustl.edu/tinker/"
+SRC_URI="https://dasher.wustl.edu/${PN}/downloads/${P}.tar.gz"
 
 SLOT="0"
 LICENSE="Tinker"
 KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
 IUSE="examples"
 
-RDEPEND="
+COMMON_DEPEND="
 	>=sci-libs/fftw-3.2.2-r1[fortran,threads]
 	dev-libs/maloc
 	!sys-apps/bar
 	!dev-util/diffuse
-	>=virtual/jre-1.6
+"
+RDEPEND="${COMMON_DEPEND}
+	>=virtual/jre-1.8:*
 "
 DEPEND="
-	${RDEPEND}
-	>=virtual/jdk-1.6
-	virtual/pkgconfig
+	${COMMON_DEPEND}
+	>=virtual/jdk-1.8:*
 "
+BDEPEND="virtual/pkgconfig "
 
 RESTRICT="mirror"
```

Comment:
Please move the `RESTRICT` to be after `IUSE` (in same block)

Reply:
> Please move the `RESTRICT` to be after `IUSE` (in same block)

done.